src/main/resources/com/mysql/cj/LocalizedErrorMessages.properties

```diff
@@ -1100,4 +1094,42 @@ ClusterAwareReaderFailoverHandler.2=[ClusterAwareReaderFailoverHandler] Connecte
 ClusterAwareReaderFailoverHandler.3=[ClusterAwareReaderFailoverHandler] Trying to connect to reader [{0,number,#}] ''{1}''
 ClusterAwareReaderFailoverHandler.4=[ClusterAwareReaderFailoverHandler] Connected to reader [{0,number,#}] ''{1}''
 ClusterAwareReaderFailoverHandler.5=[ClusterAwareReaderFailoverHandler] Failed to connect to reader [{0,number,#}] ''{1}''
-ClusterAwareReaderFailoverHandler.6=[ClusterAwareReaderFailoverHandler] {0} was called with an invalid (null or empty) topology
\ No newline at end of file
+
+ReadWriteSplittingPlugin.1=[ReadWriteSplittingPlugin] An error occurred while trying to switch to a reader connection.
```

Comment:
I think more descriptive names are better for readability in the code.

Reply:
Good idea, will do